src/DGtal/topology/CubicalComplex.ih

```diff
@@ -244,7 +244,7 @@ typename DGtal::CubicalComplex<TKSpace, TCellContainer>::Size
 DGtal::CubicalComplex<TKSpace, TCellContainer>::
 nbCells( Dimension d ) const
 {
-  return myCells[ d ].size();
+  return static_cast<typename DGtal::CubicalComplex<TKSpace, TCellContainer>::Size>(myCells[ d ].size());
```

Comment:
Yup... warnings are becoming extreme on Xcode

Reply:
Huh... and do you have the warning is that specific case (with the involved types)? Just to check because I just realized that `NumberTraits` may not work with types like `long long int` because it is not always an alias of a `intXX_t` type like (I will open an issue).